src/client/components/pages/entities/cb-review.js

```diff
@@ -78,6 +78,7 @@ class EntityReviews extends React.Component {
 		};
```

Comment:
On line 63, I'm not sure we need the `pb-3` class; it's making the padding at the top and bottom uneven